browser/src/shared/code-hosts/github/fileInfo.ts

```diff
@@ -41,7 +41,8 @@ export const resolveFileInfo = (): BlobInfo => {
     const { revisionAndFilePath, rawRepoName } = parsedURL
 
     const filePath = getFilePath()
-    const filePathWithLeadingSlash = filePath.startsWith('/') ? filePath : `/${filePath}`
+    // Don't prepend empty filePath with slash; it's the root directory of the repo
+    const filePathWithLeadingSlash = !filePath ? '' : filePath.startsWith('/') ? filePath : `/${filePath}`
```

Comment:
```suggestion
    const filePathWithLeadingSlash = filePath && !filePath.startsWith('/') ? `/${filePath}` : filePath
```

Nit: I would find this slightly easier to grok with a single ternary operator.